app/scripts/modules/core/src/reactShims/react.injector.ts

```diff
@@ -79,6 +84,7 @@ export class CoreReactInject extends ReactInject {
   public get loadBalancerFilterService() { return this.$injector.get('loadBalancerFilterService') as LoadBalancerFilterService; }
   public get manualJudgmentService() { return this.$injector.get('manualJudgmentService') as ManualJudgmentService; }
   public get modalService() { return this.$injector.get('$uibModal') as IModalService; }
+  public get MultiselectModel() { return this.$injector.get('MultiselectModel') as any; }
```

Comment:
I hope you feel dirty putting the first `any` in this file instead of converting it to TS first.